packages/agreement-email-sender/Dockerfile

```diff
@@ -11,7 +11,6 @@ COPY ./packages/agreement-email-sender/package.json /app/packages/agreement-emai
 COPY ./packages/commons/package.json /app/packages/commons/package.json
 COPY ./packages/models/package.json /app/packages/models/package.json
 COPY ./packages/kafka-iam-auth/package.json /app/packages/kafka-iam-auth/package.json
-COPY ./packages/selfcare-v2-client/package.json /app/packages/selfcare-v2-client/package.json
```

Comment:
After updating the branch, you will also find `api-clients` here. Could you delete that as well?